pkg/util/vpn.go

```diff
@@ -0,0 +1,129 @@
+package util
```

Comment:
We can probably create a "vpn" package and not have random stuff added to the "utils" package.